docs/zkapps/tutorials/anonymous-message-board.mdx

```diff
@@ -2,187 +2,198 @@
 
-[How to write a zkApp](/zkapps/how-to-write-a-zkapp) provides a high-level overview of everything you need to know to build applications
-on Mina. This tutorial will put these ideas into practice as we walk through the
-design and implementation of a semi-anonymous messaging protocol.
-
-## Overview
-
-We’ll build a smart contract that allows users to publish messages semi-anonymously. Our contract will allow a specific set of users to create new messages but will not disclose which user within the set has done so. This way, people can leverage one aspect of their identity without revealing exactly who they are. So, for example, a DAO member could make credible statements on behalf of their DAO without revealing their specific individual identity.
-
-## Setup
+This example shows you how to put building zkApp ideas into practice as you walk through designing and implementing of a semi-anonymous messaging protocol.
 
-First, install the [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) if you haven’t already done so.
+In this tutorial, you build a smart contract that allows users to publish messages semi-anonymously. 
+- The contract allows a specific set of users to create new messages but does not disclose which user creates the message. 
+- This semi-anonymous messaging leverages one aspect of a person's identity without revealing exactly who they are. 
 
-#### Dependencies
+An example use case for this semi-anonymous contract is to enable a DAO member to make credible statements on behalf of their DAO without revealing their specific individual identity.
 
-You'll need the following installed to use the zkApp CLI:
+## Prerequisites
 
-- NodeJS 16+ (or 14 using `--experimental-wasm-threads`)
-- NPM 6+
-- Git 2+
+Ensure your environment meets the [Prerequisites](/zkapps/tutorials#prerequisites) for zkApp Developer Tutorials.
 
-If you have an older version installed, we suggest installing a newer version
-using the package manager for your system: [Homebrew](https://brew.sh/) (Mac),
-[Chocolatey](https://chocolatey.org/) (Windows), or apt/yum/etc (Linux). On
-Linux, you may need to install a recent NodeJS version via NodeSource
-([deb](https://github.com/nodesource/distributions#debinstall) or
-[rpm](https://github.com/nodesource/distributions#rpminstall)), as recommended
-by the NodeJS Project.
-
-#### Installation
+In particular, make sure you have the zkApp CLI installed:
 
 ```sh
 $ npm install -g zkapp-cli
 ```
 
-#### Usage
-
-```sh
-$ zk --help
-```
-
-##### Create a new project
-
-```sh
-$ zk project message-board  # or path/to/message-board
-
-✔ Fetch project template
-✔ Initialize Git repo
-✔ NPM install
-✔ Set project name
-✔ Git init commit
-
-Success!
+This tutorial has been tested with:
 
-Next steps:
-  cd message-board
-  git remote add origin <your-repo-url>
-  git push -u origin main
-```
+- [zkApp CLI](https://github.com/o1-labs/zkapp-cli) version 0.17.2
+- [o1js](https://www.npmjs.com/package/o1js) version 0.16.2
```

Comment:
I'm commenting to leave a note that both `o1js` and the `zkApp-CLI` versions will need to be updated once the next release is out which will incorporate breaking changes.